src/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface FilterConfig {
```

Comment:
In the description of the GPC, you mention the use of cucumber-tag-expressions, but you seem to expect individual tags and not the expression. I'd suggest updating the implementation to use the expression (https://www.npmjs.com/package/cucumber-tag-expressions).